### DISCUSSION TITLE
Using stencil apps in plain HTML

### DISCUSSION THREAD

**Discussion by sunny-ps** [nxext/nx-extensions#655] [Q&A] [answered]
Hi. Firstly, thanks for the plugin. I have been working on a stencil design system, and so far integration with React has been smooth. How would I go about integrating my components into a plain HTML document?

In a normal stencil project, you would publish the package to NPM or use something like `npm link`  and then use the `<script />` to consume the components. How would I go about doing this in a Nx monorepo?

**Comment by DominikPieper** [comment, marked as answer]
Good morning,
the problem here is, that it won' work without any bundling technology. Nx works on the top of typescript paths. For a local just html project, you'd need to add the relative path  to the dist folder and look where you put the artifacts for deployment yourself.

**Reply by sunny-ps** [reply]
That was what I eventually came to understand. The problem is that unlike a stencil app or stencil design system, each app is somewhat "isolated". It doesn't have access to the the parent directory. But I imagine we could work around that by tweaking the `project.json` or or setting up webpack. Thanks for your reply. I'll mark this as the answer in the meantime.
